src/main/scala/org/ergoplatform/modifiers/ErgoFullBlock.scala

```diff
@@ -38,8 +38,8 @@ case class ErgoFullBlock(header: Header,
 
   override lazy val size: Int = header.size + blockTransactions.size + adProofs.map(_.size).getOrElse(0)
 
-  override lazy val serializer: Serializer[ErgoFullBlock] = throw new Error("Never try to serialize ErgoFullBlock")
-
+  override lazy val serializer: Serializer[ErgoFullBlock] =
```

Comment:
This message means, that it would be good to implement serialization of ErgoFullBlock, I'd like to mention here, that it is not implemented by design, not because of lack of time

Reply:
"Never try" is not informative message then, as the first reasonable question about that is "why?", and there is no any answer around. Changed message a little bit.